src/pages/OnboardingEmployees/BaseOnboardingEmployees.tsx

```diff
@@ -43,6 +43,20 @@ function BaseOnboardingEmployees({shouldUseNativeStyles, route}: BaseOnboardingE
         });
     }, [translate, selectedCompanySize]);
 
+    const [allPersonalDetails] = useOnyx(ONYXKEYS.PERSONAL_DETAILS_LIST);
+
+    const setOptimticQAGuidePersonalDetail = () => {
```

Comment:
This too is for the optimistic generation of report actions because `CreateWorkspace`'s response still does not have the personal detail value for `qa.guide`. This can be omitted if we use `Concierge` or `currentUserAccountID` as the actor for the optimistic report actions. The actor is overwritten with `qa.guide` in backend's response.